speech/auto_punctuation/auto_punctuation.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command auto_punctuation sends audio data to the Google Speech API
+// and prints its transcript with auto punctuation enabled.
+package main
```

Comment:
Does this need to be a binary? Otherwise, the usual guidance is to have standalone snippets so we don't have to maintain the rest of the binary.

Reply:
Alright, so I removed the main and just left the `autoPunctuation` function. I also updated the READMEs to only mention the docs. Thank you!